test/cashu/secret.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, test } from 'bun:test';
+import { parseSecret } from '../../lib/cashu/secret';
+
+describe('parseSecret', () => {
+  test('should return a secret as described in NUT00', () => {
```

Comment:
I would maybe rephrase this to state something like `should return the input value if input value is NUT00 secret`

Reply:
done